cmd/seed/main.go

```diff
@@ -2,138 +2,134 @@ package main
+	// Create a single default task
+	title := "Default Mock Task"
+	expireDuration := 6 * time.Hour
+	if _, err := fixtureService.CreateDefaultTask(ctx, title, expireDuration); err != nil {
+		log.Error().Err(err).Msg("Failed to create default task")
 		return
 	}
 
-	var taskData map[string]interface{}
-	err = json.Unmarshal(byteValue, &taskData)
-	if err != nil {
-		log.Error().Err(err).Msg("Failed to unmarshal task data")
-		return
+	log.Info().Msg("Reset MinerUser and created a single default task successfully")
+}
+
+// Function to generate expired tasks
+func generateExpiredTasks(client *db.PrismaClient, ctx context.Context) {
+	fixtureService := fixtures.NewFixtureService(client)
+
+	for i := 0; i < 3; i++ {
+		title := "Expired Task"
+		expireDuration := -6 * time.Hour
+		if _, err := fixtureService.CreateDefaultTask(ctx, title, expireDuration); err != nil {
+			log.Error().Err(err).Msg("Failed to create expired task")
+			return
+		}
 	}
 
-	// Convert taskData to types.JSON
-	taskDataJSON, err := json.Marshal(taskData)
-	if err != nil {
-		log.Error().Err(err).Msg("Failed to marshal task data")
-		return
+	log.Info().Msg("Expired tasks created successfully")
+}
+
+// Function to generate tasks with short expiration
+func generateShortExpireTasks(client *db.PrismaClient, ctx context.Context) {
+	fixtureService := fixtures.NewFixtureService(client)
+
+	for i := 0; i < 3; i++ {
+		title := "Task with Short Expiration"
+		expireDuration := 5 * time.Minute
+		if _, err := fixtureService.CreateDefaultTask(ctx, title, expireDuration); err != nil {
+			log.Error().Err(err).Msg("Failed to create short expiration task")
+			return
+		}
 	}
 
-	// Add create transaction for the new task
-	txns = append(txns, client.Task.CreateOne(
-		db.Task.ExpireAt.Set(time.Now().Add(24*7*time.Hour)),
-		db.Task.Title.Set("Mock Task1"),
-		db.Task.Body.Set("This is a sample task body"),
-		db.Task.Type.Set(db.TaskTypeCodeGeneration),
-		db.Task.TaskData.Set(types.JSON(taskDataJSON)),
-		db.Task.Status.Set(db.TaskStatusInProgress),
-		db.Task.MaxResults.Set(10),
-		db.Task.NumResults.Set(0),
-		db.Task.NumCriteria.Set(0),
-		db.Task.TotalReward.Set(101.0),
-		db.Task.MinerUser.Link(
-			db.MinerUser.Hotkey.Equals(mockHotKey),
-		),
-	).Tx())
-
-	// Execute all transactions
-	if err := client.Prisma.Transaction(txns...).Exec(ctx); err != nil {
-		log.Error().Err(err).Msg("Error executing transaction")
-		return
+	log.Info().Msg("Short expiration tasks created successfully")
+}
+
+// Function to generate tasks with normal expiration
+func generateNormalExpireTasks(client *db.PrismaClient, ctx context.Context) {
+	fixtureService := fixtures.NewFixtureService(client)
+
+	for i := 0; i < 3; i++ {
+		title := "Task with Normal Expiration"
+		expireDuration := 6 * time.Hour
+		if _, err := fixtureService.CreateDefaultTask(ctx, title, expireDuration); err != nil {
+			log.Error().Err(err).Msg("Failed to create normal expiration task")
+			return
+		}
```

Comment:
do you want to simplify them into one function and specify time as an argument instead? in seconds